gigpress.php

```diff
@@ -674,6 +673,263 @@ function fetch_gigpress_venues() {
+		"MV" => "Maldives",
+		"ML" => "Mali",
+		"MT" => "Malta",
+		"MH" => "Marshall Islands",
+		"MQ" => "Martinique",
+		"MR" => "Mauritania",
+		"MU" => "Mauritius",
+		"YT" => "Mayotte",
+		"MX" => "Mexico",
+		"FM" => "Micronesia",
+		"MD" => "Moldova",
+		"MC" => "Monaco",
+		"MN" => "Mongolia",
+		"ME" => "Montenegro",
+		"MS" => "Montserrat",
+		"MA" => "Morocco",
+		"MZ" => "Mozambique",
+		"MM" => "Myanmar",
+		"NA" => "Namibia",
+		"NR" => "Nauru",
+		"NP" => "Nepal",
+		"NL" => "Netherlands",
+		"NC" => "New Caledonia",
+		"NZ" => "New Zealand",
+		"NI" => "Nicaragua",
+		"NE" => "Niger",
+		"NG" => "Nigeria",
+		"NU" => "Niue",
+		"NF" => "Norfolk Island",
+		"MP" => "Northern Mariana Islands",
+		"NO" => "Norway",
+		"OM" => "Oman",
+		"PK" => "Pakistan",
+		"PW" => "Palau",
+		"PS" => "Palestine",
+		"PA" => "Panama",
+		"PG" => "Papua New Guinea",
+		"PY" => "Paraguay",
+		"PE" => "Peru",
+		"PH" => "Philippines",
+		"PN" => "Pitcairn",
+		"PL" => "Poland",
+		"PT" => "Portugal",
+		"PR" => "Puerto Rico",
+		"QA" => "Qatar",
+		"RE" => "Reunion",
+		"RO" => "Romania",
+		"RU" => "Russian Federation",
+		"RW" => "Rwanda",
+		"BL" => "Saint Barthelemy",
+		"SH" => "Saint Helena",
+		"KN" => "Saint Kitts and Nevis",
+		"LC" => "Saint Lucia",
+		"MF" => "Saint Martin",
+		"PM" => "Saint Pierre and Miquelon",
+		"VC" => "Saint Vincent and the Grenadines",
+		"WS" => "Samoa",
+		"SM" => "San Marino",
+		"ST" => "Sao Tome and Principe",
+		"SA" => "Saudi Arabia",
+		"SN" => "Senegal",
+		"RS" => "Serbia",
+		"SC" => "Seychelles",
+		"SL" => "Sierra Leone",
+		"SG" => "Singapore",
+		"SX" => "Sint Maarten",
+		"SK" => "Slovakia",
+		"SI" => "Slovenia",
+		"SB" => "Solomon Islands",
+		"SO" => "Somalia",
+		"ZA" => "South Africa",
+		"GS" => "South Georgia and South Sandwich Islands",
+		"ES" => "Spain",
+		"LK" => "Sri Lanka",
+		"SD" => "Sudan",
+		"SS" => "South Sudan",
+		"SR" => "Suriname",
+		"SJ" => "Svalbard and Jan Mayen",
+		"SZ" => "Swaziland",
+		"SE" => "Sweden",
+		"CH" => "Switzerland",
+		"SY" => "Syria",
+		"TW" => "Taiwan",
+		"TJ" => "Tajikistan",
+		"TZ" => "Tanzania",
+		"TH" => "Thailand",
+		"TL" => "Timor-Leste",
+		"TG" => "Togo",
+		"TK" => "Tokelau",
+		"TO" => "Tonga",
+		"TT" => "Trinidad and Tobago",
+		"TN" => "Tunisia",
+		"TR" => "Turkey",
+		"TM" => "Turkmenistan",
+		"TC" => "Turks and Caicos Islands",
+		"TV" => "Tuvalu",
+		"UG" => "Uganda",
+		"UA" => "Ukraine",
+		"AE" => "United Arab Emirates",
+		"UK" => "United Kingdom",
+		"US" => "United States",
+		"UM" => "United States Minor Outlying Islands",
+		"UY" => "Uruguay",
+		"UZ" => "Uzbekistan",
+		"VU" => "Vanuatu",
+		"VE" => "Venezuela",
+		"VN" => "Vietnam",
+		"VG" => "Virgin Islands (British)",
+		"VI" => "Virgin Islands (U.S.)",
+		"WF" => "Wallis and Futuna",
+		"EH" => "Western Sahara",
+		"YE" => "Yemen",
+		"ZM" => "Zambia",
+		"ZW" => "Zimbabwe"
+	);
+
+	$gp_countries = apply_filters( 'gigpress_country_list', $gp_countries );
```

Comment:
```suggestion
	/**
	 * Filters the country list.
	 *
	 * @since TBD
	 *
	 * @param array $gp_countries The country list.
	 */
	$gp_countries = apply_filters( 'gigpress_country_list', $gp_countries );
```

Reply:
Thanks @juanfra , I just wanted to add the docblock now. :) You beat me to it.